<?xml version="1.0"?>

<launch>

  <arg name="watchdog_enabled" default="false" />

  <!-- Load joint controller configurations from YAML file to parameter server -->
  <rosparam file="$(find taurob_flipper_ros_control)/config/taurob_flipper.yaml" command="load"/>
  
  <node name="flipper_hw_interface" pkg="taurob_flipper_ros_control" type="flipper_hw_interface" ns="argonaut_robot/flipper" output="screen">
    <param name="watchdog" value="$(arg watchdog_enabled)" />
    <param name="control_enabled_at_startup" value="true" />
    <param name="ip_address" value="10.0.0.41" />
    <param name="port" value="1111" />
    <remap from="/argonaut_robot/flipper/joint_states" to="/joint_states" />
  </node>

</launch>
